h/static/styles/partials-v2/_search.scss

```diff
@@ -0,0 +1,128 @@
+// Search page
+// -----------
+// Styles for the search page which displays the results
+// of activity queries.
+
+.search-result-container {
+  font-size: $normal-font-size;
+  margin-top: 20px;
+  color: $grey-6;
+}
+
+.search-result-list {
+  list-style: none;
+  max-width: 950px;
+}
+
+.search-result-bucket {
+  padding: 10px 0px;
+  border-bottom: 1px solid $grey-3;
+}
+
+.search-result__details {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  padding-left: 0px;
+  background-color: $grey-2;
+
+  &:hover {
+    background-color: $grey-2;
+  }
+}
+
+.search-result__timeframe {
+  font-weight: bold;
+  padding-bottom: 10px;
+  border-bottom: 1px solid $grey-2;
+  margin-top: 30px;
+}
+
+.search-result__domain {
+  width: 130px;
+  text-indent: 10px;
+  color: $grey-4;
+}
+
+.search-result__title {
+  font-weight: bold;
+  max-width: 695px;
+}
+
+.search-result__annotations-count-container {
+  width: 30px;
+  padding: 5px 0px;
+  background-color: $grey-2;
+  text-align: center;
+  border-radius: 2px;
+  float: right;
+}
+
+.search-result__annotations-count {
+  font-weight: bold;
+  width: 75px;
+}
+
+.search-result__annotation-cards-info {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  margin-top: 10px;
+}
+
+.search-result__annotation-cards {
+  width: 500px;
+  margin-left: -38px;
+}
+
+.search-result__annotation-metadata {
+  @include font-normal;
+  width: 210px;
+  margin-left: 30px;
+  word-wrap: break-word;
+}
+
+.search-result__metadata-key {
+  color: $grey-4;
+}
+
+.search-result__metadata-val {
+  color: $grey-6;
+  margin-bottom: 20px;
+}
+
+@media screen and (max-width: $max-phone-width) {
```

Comment:
I find it helpful to have a description in the CSS file of how the responsive layout works from a user's perspective. See `_nav-bar.scss` or `_form-input.scss` for example.